cdf_fabric_replicator/event.py

```diff
@@ -0,0 +1,101 @@
+import logging
+import time
+
+from typing import Any, Dict, List, Iterator
+from cognite.extractorutils.base import CancellationToken
+
+from cognite.extractorutils.base import Extractor
+
+from azure.identity import DefaultAzureCredential
+from deltalake import write_deltalake
+import pyarrow as pa
+
+from cdf_fabric_replicator import __version__
+from cdf_fabric_replicator.config import Config
+from cdf_fabric_replicator.metrics import Metrics
+from cognite.client.data_classes import EventList
+
+class EventsReplicator(Extractor):
+    def __init__(self, metrics: Metrics, stop_event: CancellationToken) -> None:
+        super().__init__(
+            name="cdf_fabric_replicator_data_modeling",
+            description="CDF Fabric Replicator",
+            config_class=Config,
+            metrics=metrics,
+            use_default_state_store=False,
+            version=__version__,
+            cancellation_token=stop_event,
+        )
+        self.metrics: Metrics
+        # self.stop_event = stop_event
```

Comment:
does this comment still need to be here?

Reply:
removed